deploy/pivot_role/pivotRoleCDK/dataall_base_infra.py

```diff
@@ -0,0 +1,827 @@
+                                             "lakeformation:AddLFTagsToResource",
+                                             "lakeformation:RemoveLFTagsFromResource",
+                                             "lakeformation:GetResourceLFTags",
+                                             "lakeformation:ListLFTags",
+                                             "lakeformation:CreateLFTag",
+                                             "lakeformation:GetLFTag",
+                                             "lakeformation:UpdateLFTag",
+                                             "lakeformation:DeleteLFTag",
+                                             "lakeformation:SearchTablesByLFTags",
+                                             "lakeformation:SearchDatabasesByLFTags",
+                                             "lakeformation:ListResources",
+                                             "lakeformation:ListPermissions",
+                                             "lakeformation:GrantPermissions",
+                                             "lakeformation:BatchGrantPermissions",
+                                             "lakeformation:RevokePermissions",
+                                             "lakeformation:BatchRevokePermissions",
+                                             "lakeformation:PutDataLakeSettings",
+                                             "lakeformation:GetDataLakeSettings",
+                                             "lakeformation:GetDataAccess",
+                                             "lakeformation:SearchTablesByLFTags",
+                                             "lakeformation:SearchDatabasesByLFTags",
+                                             "lakeformation:GetWorkUnits",
+                                             "lakeformation:StartQueryPlanning",
+                                             "lakeformation:GetWorkUnitResults",
+                                             "lakeformation:GetQueryState",
+                                             "lakeformation:GetQueryStatistics",
+                                             "lakeformation:StartTransaction",
+                                             "lakeformation:CommitTransaction",
+                                             "lakeformation:CancelTransaction",
+                                             "lakeformation:ExtendTransaction",
+                                             "lakeformation:DescribeTransaction",
+                                             "lakeformation:ListTransactions",
+                                             "lakeformation:GetTableObjects",
+                                             "lakeformation:UpdateTableObjects",
+                                             "lakeformation:DeleteObjectsOnCancel",
+                                             "lakeformation:RegisterResource"
+                                         ],
```

Comment:
Missing `'lakeformation:DescribeResource'` to match pivotRole yaml

Reply:
Added! Also
`"lakeformation:SearchTablesByLFTags"` and `"lakeformation:SearchDatabasesByLFTags"` where duplicated